client/src/components/OnboardingParent/OnboardingParent.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import "./OnboardingParent.css";
+import logo from "../../../../assets/images/login_logo.png";
+import CarouselDots from "../CarouselDots/CarouselDots";
+// Class Based React Component
+class OnboardingParent extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "OnboardingParent"
+    };
+  }
+
+  // Runs after Component is loaded in the broswer
+  componentDidMount() {}
+
+  // Runs after a component has been updated
+  componentDidUpdate() {}
+
+  // Runs right before a component is removed from the DOM
+  componentWillUnmount() {}
```

Comment:
You don't need lifecycle methods if you don't have functionality that you specifically want to attach to a stage in the component lifecycle. Think about it this way - if you're not using them, you don't need to include them.